lib/tasks/db.js

```diff
@@ -41,6 +41,38 @@ module.exports = {
     }
   },
 
+  'db:migrate:pending': {
+    descriptions: {
+      'short': 'List pending migrations.',
+      'long': [
+        'The command lists pending migrations and exits with non-zero status if',
+        'there are any.'
+      ]
+    },
+
+    task: function () {
+      return getMigrator('migration').then(function (migrator) {
+        return ensureCurrentMetaSchema(migrator).then(function () {
+          return migrator.pending();
+        }).then(function (migrations) {
+          if (migrations.length === 0) {
+            console.log('No pending migrations, database schema up to date.');
+            process.exit(0);
+          } else {
+            console.log('Pending migrations:');
+            migrations.forEach(function (migration) {
+              console.log(migration.file);
+            });
+            process.exit(1);
+          }
+        }).catch(function (err) {
+          console.error(err);
+          process.exit(1);
```

Comment:
`process.exit(255);`?
I wouldn't exit with the same exit code as if it were pending migrations.